src/custom.fixtures.js

```diff
@@ -0,0 +1,34 @@
+import { TEST_FIXTURES, ROOT_FINGERPRINT } from "unchained-bitcoin";
```

Comment:
I was starting to do this in the config branch, but what do you think about having a fixtures (or sometimes I've seen it named `data`) folder? I imagine there will only be more of these as time goes on and could help keep things organized.

Reply:
I could go either way, I think given we already have
`name`
`name.test`

adding in
`name.fixtures` seems fine.

but also given the scale this could reach, then the TLD gets pretty crowded so could also see 
`data`
`data/name.fixtures` as well.

are you advocating only moving fixtures, or also trying to consolidate where tests live?

Reply:
Only moving fixtures